dao/src/main/kotlin/Database.kt

```diff
@@ -17,7 +17,7 @@
  * License-Filename: LICENSE
  */
 
-@file:Suppress("TooManyFunctions")
+@file:Suppress("TooManyFunctions", "TooGenericExceptionCaught")
```

Comment:
To avoid this, maybe below use

    return runCatching {
        HikariDataSource(dataSourceConfig)
    }.onFailure {
        logger.error("Failed to create data source.", it)
    }.getOrThrow()

Reply:
Good idea.